client/src/components/events/like.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { HeartOutlined, HeartFilled } from '@ant-design/icons'
+import { FlexContainer } from 'cornell-glue-ui'
+
+import { IEvent } from 'src/types/event.type'
+
+// interface AvatarProps extends React.HTMLAttributes<HTMLDivElement> {
+//   name: string
+//   avatarColor?: string
+// }
+
+interface LikeProps {
+  event: IEvent, 
+}
+
+const LikeContainer:React.FC<LikeProps> = ({event}) => {
```

Comment:
generally speaking, the file name should be identical to the React component that's being exported by default. so the file should be named something like LikeContainer.tsx

maybe LikeButton.tsx could be a more accurate name?